public/javascripts/helpers.js

```diff
@@ -0,0 +1,36 @@
+function ajax(method, url, callback) {
+    var xmlhttp;
+
+    if (window.XMLHttpRequest) {
+        // code for IE7+, Firefox, Chrome, Opera, Safari
+        xmlhttp = new XMLHttpRequest();
+    } else {
+        // code for IE6, IE5
+        xmlhttp = new ActiveXObject("Microsoft.XMLHTTP");
```

Comment:
do we really need to support ie5 and 6. like we r gonna run this on somethign with webkit or firefox